jenkins/e2e-image/Dockerfile

```diff
@@ -53,9 +65,7 @@ RUN tar xzf /workspace/google-cloud-sdk.tar.gz -C / && \
         --bash-completion=false \
         --path-update=false \
         --usage-reporting=false && \
-    gcloud components install beta && \
-    gcloud components install alpha && \
-    gcloud components update kubectl && \
+    gcloud components install alpha beta kubectl && \
```

Comment:
FYI I also updated this to install all three components in one shot - it speeds up this step a bit.